analyses/snv-callers/scripts/03-calculate_tmb.R

```diff
@@ -146,6 +154,19 @@ join_cols <- c(
   "Tumor_Sample_Barcode"
 )
 
+# Variant Classification with High/Moderate variant consequences from maftools
+nonsynonymous <- c(
+  "Missense_Mutation",
+  "Frame_Shift_Del",
+  "In_Frame_Ins",
+  "Frame_Shift_Ins",
+  "Splice_Site",
+  "Nonsense_Mutation",
+  "In_Frame_Del",
+  "Nonstop_Mutation",
+  "Translation_Start_Site"
+)
```

Comment:
As we have discussed, this does differ slightly from the standard proposed in https://jitc.bmj.com/content/8/1/e000147#DC by the *Friends of Cancer Research TMB Harmonization Project*. 

The relevant table is here: https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/725#issuecomment-657575733

In particular, the FoCR group does not include `Translation_Start_Site`, `Nonstop_Mutation` or `Splice_Site`. I tend to think all of those are worth including, but I guess one simple question is how common these mutations actually are in the data set. Either way, we will want to standardize our criteria across analyses.

Reply:
I think this question is something I can ask in a next PR that does some more exploration. I'll file an issue that lays out what kinds of exploration analyses I think should be included.

Reply:
Filed the issue: https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/729